compiam/melody/pitch_extraction/melodia.py

```diff
@@ -134,3 +135,14 @@ def normalise_pitch(pitch, tonic, bins_per_octave=120, max_value=4):
         return normalisation(
             pitch, tonic, bins_per_octave=bins_per_octave, max_value=max_value
         )
+
+    @staticmethod
+    def write_csv(data, output_path):
```

Comment:
This is not required. Just use the original function?

Reply:
hhhhmmm but how? Because we need to have a callable function into FTANet somewhere, right? Or would you add that as optional inside `predict`, and then trigger it with an additional parameter `save_output` for instance. However, that would be a bit tedious, since we would need to also add an additional `output_dir` or `output_path` parameter.